src/index.js

```diff
@@ -20,9 +22,13 @@ ReactDOM.render(
         <ThemeProvider>
           <VideosProvider>
             <FilterProvider>
-              <SidebarProvider>
-                <App />
-              </SidebarProvider>
+              <WatchLaterVideosProvider>
+                <LikedVideosProvider>
+                  <SidebarProvider>
+                    <App />
+                  </SidebarProvider>
+                </LikedVideosProvider>
+              </WatchLaterVideosProvider>
             </FilterProvider>
           </VideosProvider>
         </ThemeProvider>
```

Comment:
You could make a combined provider and wrap only that around.

Reply:
Will look into this. Thanks!